src/java/spring/build.gradle

```diff
@@ -0,0 +1,58 @@
+plugins {
+    id 'java'
+    id 'org.springframework.boot' version '3.2.4'
+    id 'io.spring.dependency-management' version '1.1.4'
+}
+
+group = 'app.planentnine'
+version = '0.0.1-SNAPSHOT'
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_21
+    targetCompatibility = JavaVersion.VERSION_21
+
+    toolchain {
+        languageVersion= JavaLanguageVersion.of(21)
+    }
+}
+
+configurations {
+    compileOnly {
+        extendsFrom annotationProcessor
+    }
+}
+
+repositories {
+    mavenCentral()
+    mavenLocal()
+}
+
+ext {
+    liquibaseVersion = '4.24.0'
+    lombokVersion = '1.18.32'
+    mybatisSpringBootVersion = '3.0.2'
+    postgresqlVersion = '42.6.0'
+    springBootVersion = '3.2.4'
+}
+
+dependencies {
+    //Remove with Sessionless
```

Comment:
somewhat, sessionless protocol needs to be placed somewhere as its own artifact hosted. but since it currently isnt im just leaving it embedded in the stack

Reply:
ah gotcha. We can send it to maven when we're ready.